python/ray/tune/tune.py

```diff
@@ -90,7 +90,8 @@ def run(run_or_experiment,
         queue_trials=False,
         reuse_actors=False,
         trial_executor=None,
-        raise_on_failed_trial=True):
+        raise_on_failed_trial=True,
+        ray_redis_address="auto"):
```

Comment:
Do we need to handle redis address too? It's probably easier to only handle the local init case, which can be done without passing in any extra args in `tune.run` only.

Reply:
Done